R/zzz.R

```diff
@@ -5,41 +5,17 @@
 #' @noRd
 #' @keywords internal
 .onAttach <- function(...) {
-  # if(!valid_githubpat())
```

Comment:
`daapr` admittedly doesn't need `GITHUB_PAT`, but do we want to maintain a warning when loading `dpbuild`? I can't recall if we have the warning there.

Reply:
@amashadihossein `dpbuild` does check for `GITHUB_PAT` in `dpconf_get()` and `dp_push()`. Let me know if that sounds sufficient. The `GITHUB_PAT` checking in daapr has also been commented out for a while.

Reply:
> @amashadihossein `dpbuild` does check for `GITHUB_PAT` in `dpconf_get()` and `dp_push()`. Let me know if that sounds sufficient. The `GITHUB_PAT` checking in daapr has also been commented out for a while.

Perfect! Leaving it to functions that require communication w/ GitHub is sufficient.  Thinking of this as a configuration validation (and the sort of things for which it is better to fail early in the workflow than later), it is helpful that `dpconf_get`, which is called early on, checks this. Thanks for pointing that out!